bin/receiver.py

```diff
@@ -47,6 +47,9 @@ def main():
 
     (options, unused_args) = op.parse_args()
 
+    # Absolute file path required when refreshing dn_file, relative path resulted in an error. 
+    options.dn_file=os.path.abspath(options.dn_file)
```

Comment:
Spaces around operators is the usual Python style. 

```suggestion
    options.dn_file = os.path.abspath(options.dn_file)
```